core/Updates/2.15.1-b1.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Common;
+use Piwik\Updater;
+use Piwik\Updates;
+
+
+class Updates_2_15_1_b1 extends Updates
+{
+    public function getMigrationQueries(Updater $updater)
+    {
+        $updateSql = array(
+            'ALTER TABLE `' . Common::prefixTable('user_language')
```

Comment:
Should we maybe put this update into `UserLanguage` plugin as the table is defined there as well?